### ISSUE TITLE
Revision of relationship of GO:0060468 (prevention of polyspermy) and GO:0060471 ( cortical granule exocytosis)

### ISSUE THREAD

**Issue by pgarmiri** [geneontology/go-ontology#13980]
Hi,
I came across this as part of the analysis for the signaling pathway for fertilization.

The current relationship of the terms seems to be that:  GO:0060471 ( cortical granule exocytosis) is part_of GO:0060468 (prevention of polyspermy).
(see the tree below)
http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0060471#term=ancchart

The relationship   between the two terms might need to be updated as the prevention of  polyspermy is the result of colrtical reaction (cortical granule exocytosis). (See below for some definitions/ references supporting that.)

Based on those definitions, I would also suggest considering adding the synonym "Cortical reaction" to  GO:0060471 ( cortical granule exocytosis).

Cortical   reaction - Wikipedia
 
https://en.wikipedia.org/wiki/Cortical_reaction
The cortical reaction is a process   initiated during fertilization by the release of cortical granules from the egg, which prevents polyspermy, the fusion of multiple sperm with one egg.
 
Fertilization   - vivo.colostate.edu
 
www.vivo.colostate.edu/hbooks/pathphys/reprod/fert/fert.html
1   Apr 2000 - The cortical reaction   refers to a massive exocytosis of cortical granules seen shortly after sperm-oocyte fusion.  Cortical granules contain a mixture   of enzymes, including several proteases, which diffuse into the zona   pellucida following exocytosis from the egg.

Thanks,

Penelope

**Comment by ukemi** [comment]
I think there are two phases to the polyspermy block (at least in sea urchins). There is a fast block which is electrical and a slow block which is physical.

**Comment by pgarmiri** [comment]
Hi David, I see what you mean.  Only the Slow block of polyspermy is the result of the cortical reaction, which seems to be specific to mammals according to wikipedia. (https://en.wikipedia.org/wiki/Polyspermy)

Perhaps the two phases should have their own term with a taxon restrain for the slow block one? If only that term was linked with the GO:0060471 ( cortical granule exocytosis), would that make sense?

**Comment by ukemi** [comment]
I'm not sure that the slow block is specific to mammals. Going back to my ancient history here, in urchins the cortical granule reaction results in the establishment of the fertilization envelope. 
PMID:2045963

**Comment by pgarmiri** [comment]
You are totally right! I would update my suggestion to add the two children terms (without taxon constrains) and link the cortical granule reaction only with the Slow block one.  
Urchin protein could be annotated with both terms depending on the phase they are involved in. Mammalian protein would be only annotated with the Slow block polyspermy which is what is found in the literature. 
Would that work?

**Comment by ukemi** [comment]
It might be better to describe them with respect to their mechanism, although embryologists call them fast and slow, maybe in GO it would be better to use electrical and physical/mechanical. There might be other mechanisms out there that I have forgotten about. @vanaukenk what are your thoughts?

**Comment by ukemi** [comment]
I also wonder if we are expanding the project too far and should just focus on the signaling aspects of fertilization for now.

**Comment by pgarmiri** [comment]
I would go for a descriptive name as well. The fast and slow mechanisms could perhaps be added as synonyms. Other mechanisms could be added later on. But with two it would be enough to make the distinction and link GO:0060471 ( cortical granule exocytosis) only with the physical/mechanical one.

I think, GO:0060471 ( cortical granule exocytosis) should be higher/before the physical/mechanical prevention of  polyspermy. 
If this is the case, it gets a bit complicated with the hierarchy of the terms.  It might be needed the "electrical prevention of polyspermy" to be direct child of  GO:0007343 egg activation. The "physical/mechanical prevention of  polyspermy" would be a child of GO:0060471 cortical granule exocytosis.

I know they are the last steps of the pathway as we defined it but, I think, it would be nice if the two terms are in the right order/ relationship eventually.

**Comment by ukemi** [comment]
Thinking a lot about embryology today and am now remembering that there is another block. If I remember correctly, in some organisms there is a small pore in the egg called a micropyle that only allows one sperm to enter and is then blocked. There is a lot of info about this in plants, but can't easily find anything to confirm my memory about animals.

**Comment by pgaudet** [comment]
Hi @pgarmiri @ukemi 
Do we have action items here ? 

The proposal is to create 
- "electrical prevention of polyspermy" as a direct child of GO:0007343 egg activation.
and 
- physical/mechanical prevention of polyspermy as a child of GO:0060471 cortical granule exocytosis (is this right ? prevention of polyspermy would be is_a exocytosis?) 

- Also, can you provide definitions and references ? 

Thanks, Pascale

**Comment by ukemi** [comment]
If I remember correctly, cortical granule exocytosis would be a part of the prevention of polyspermy. Prevention of polyspermy is definitely not a type of exocytosis. Off the top of my head, there are at least three mechanisms of prevention of polyspermy. The fast block is electrical where the membrane potential of the egg changes once a sperm binds. The slow block is the subsequent physical block of sperm binding to the egg. In sea urchins, this is the results of the creation of the fertilization envelope that lifts off the surface of the egg and sweeps away any excess sperm. The third way is also a mechanical block where there is a small structure called a micropyle in the egg that a sperm enters. Once a sperm enters, it plugs up the micropyle and prevents further sperm from entering.
